compiler/luci/pass/src/ConvertNCHWToNHWCPass.cpp

```diff
@@ -499,6 +499,24 @@ class ConvertNCHWToNHWC final : public luci::CircleNodeMutableVisitor<bool>
 
     return true;
   }
+
+  bool visit(luci::CircleRelu6 *node)
+  {
+    const auto pred_node = loco::must_cast<luci::CircleNode *>(node->features());
+    auto pre_trans = create_pre_transpose(node);
```

Comment:
This does not check any condition if there was exsiting Transpose node before `RELU6`. Is this OK?

Reply:
> Is this OK?

Yes. We do not check previously created Transpose nodes because Transpose is inserted only once (DataFormat annotation is checked before entering this visitor).